KsApi/models/Reward+ManagePledgeView.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+
+public extension Reward {
+  /**
+   Create an add-on reward from a ManagePledgeViewBackingEnvelope.Backing.Reward
+
+    - parameter backingReward: The ManagePledgeViewBackingEnvelope.Backing.Reward data structure.
+    - parameter project: The associated Project model.
+    - parameter selectedAddOnQuantities: The selected quantity for this add-on.
+    - parameter dateFormatter: A DateFormatter configured with the format "yyyy-MM-DD".
+
+    - returns: A Reward.
+   */
+
+  static func addOnReward(
+    from backingReward: ManagePledgeViewBackingEnvelope.Backing.Reward,
+    project: Project,
+    selectedAddOnQuantities: [String: Int],
+    dateFormatter: DateFormatter
+  ) -> Reward? {
+    guard let rewardId = decompose(id: backingReward.id) else { return nil }
+
+    let estimatedDeliveryOn = backingReward.estimatedDeliveryOn
+      .flatMap(dateFormatter.date(from:))?.timeIntervalSince1970
+
+    let addOnData = AddOnData(
+      isAddOn: true,
+      selectedQuantity: selectedAddOnQuantities[backingReward.id] ?? 0
+    )
+
+    return Reward(
+      addOnData: addOnData,
+      backersCount: backingReward.backersCount,
+      convertedMinimum: calculateConvertedMinimum(from: backingReward, with: project),
+      description: backingReward.description,
+      endsAt: backingReward.endsAt,
+      estimatedDeliveryOn: estimatedDeliveryOn,
+      id: rewardId,
+      limit: backingReward.limit,
+      minimum: backingReward.amount.amount,
+      remaining: backingReward.remainingQuantity,
+      rewardsItems: rewardItemsData(from: backingReward, with: project),
+      shipping: shippingData(from: backingReward),
+      startsAt: backingReward.startsAt,
+      title: backingReward.name
+    )
+  }
+}
+
+private func calculateConvertedMinimum(
+  from backingReward: ManagePledgeViewBackingEnvelope.Backing.Reward,
+  with project: Project
+) -> Double {
+  let rate = project.stats.currentCurrencyRate ?? project.stats.staticUsdRate
+
+  return Double(Int(ceil(Float(backingReward.amount.amount) * rate)))
```

Comment:
Any chance they could send you this value from the graphQL API? 😬 

I know we did some work on the `v1` `Reward` model to have the backend send converted minimum amounts (`converted_minimum`), would be nice to keep it consistent.

Reply:
Yeah we're going to schedule back-end work to address this, this is just for the interim 😅